v2/proto/spiffe/workload/workload.pb.go

```diff
@@ -9,6 +9,8 @@ import (
 	proto "github.com/golang/protobuf/proto"
 	_struct "github.com/golang/protobuf/ptypes/struct"
 	grpc "google.golang.org/grpc"
+	codes "google.golang.org/grpc/codes"
```

Comment:
Was this file updated intentionally? If so, shouldn't the workload api proto be updated as well?